arm-powerbiembedded/2016-01-29/swagger/powerbiembedded.json

```diff
@@ -220,12 +219,6 @@
         "responses": {
           "202": {
```

Comment:
As this operation is not long running, I think the response should be 200 rather than 202 (Accepted)